XFAIL sqlite3 tests that use temp tables

From: David Rothenberger <daveroth@acm.org>

The sqlite3 library (3.7.16.2-1) has a known problem with temporary
tables, and the wc-queries-test tests rely on this, so XFAIL them.
---
 subversion/tests/libsvn_wc/wc-queries-test.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/subversion/tests/libsvn_wc/wc-queries-test.c b/subversion/tests/libsvn_wc/wc-queries-test.c
index cd55a1c..c1f01f8 100644
--- a/subversion/tests/libsvn_wc/wc-queries-test.c
+++ b/subversion/tests/libsvn_wc/wc-queries-test.c
@@ -712,10 +712,10 @@ struct svn_test_descriptor_t test_funcs[] =
     SVN_TEST_NULL,
     SVN_TEST_PASS2(test_sqlite_version,
                    "sqlite up-to-date"),
-    SVN_TEST_PASS2(test_parsable,
-                   "queries are parsable"),
-    SVN_TEST_PASS2(test_query_expectations,
-                   "test query expectations"),
+    SVN_TEST_XFAIL2(test_parsable,
+                    "queries are parsable"),
+    SVN_TEST_XFAIL2(test_query_expectations,
+                    "test query expectations"),
     SVN_TEST_PASS2(test_schema_statistics,
                    "test schema statistics"),
     SVN_TEST_NULL
